internal/s3storage/s3events.go

```diff
@@ -0,0 +1,52 @@
+package s3storage
+
+import (
+	"encoding/json"
+)
+
+// S3Event describes events received from S3 notification service.
+// They are, kind of, documented here: https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html
+type S3Event struct {
+	Records []S3EventRecord `json:"Records,omitempty"`
+}
+
+func (p *S3Event) Unmarshal(data []byte) error {
+	return json.Unmarshal(data, p)
+}
+
+type S3EventRecord struct {
+	EventVersion      string                `json:"eventVersion,omitempty"`
+	EventSource       string                `json:"eventSource,omitempty"`
+	AwsRegion         string                `json:"awsRegion,omitempty"`
+	EventTime         string                `json:"eventTime,omitempty"`
+	EventName         string                `json:"eventName,omitempty"`
+	UserIdentity      S3EventRecordIdentity `json:"userIdentity,omitempty"`
+	RequestParameters map[string]string     `json:"requestParameters,omitempty"`
+	ResponseElements  map[string]string     `json:"responseElements,omitempty"`
+	S3                S3EventRecordData     `json:"s3,omitempty"`
+}
+
+type S3EventRecordIdentity struct {
+	PrincipalId string `json:"principalId,omitempty"` //"AWS:AROAIVBPOFACJXCN6JNTG:eki@lunar.app"
```

Comment:
There are multiple trailing comments in the structs that smell like testing output. Can we make them either more general if they need to be here, ie. remove Lunar and EKI, or can they be removed entirely?

Reply:
Its and old leftover comment. Let's remove it 👍